src/oic/utils/http_util.py

```diff
@@ -437,12 +438,18 @@ def __init__(self, srv, ttl=5):
         self.pad_chr = " "
 
     def init_srv(self, srv):
-        if srv:
-            self.srv = srv
-
-            for param in ["seed", "iv"]:
-                if not getattr(srv, param, None):
-                    setattr(srv, param, rndstr().encode("utf-8"))
+        if not srv:
+            return
```

Comment:
I'd actually be happier with making this a separate ticket if you don't mind @schlenk. I have preserved the original behaviour (I think!) in this change but your recommendation (which I think is totally valid!) will bring something new. I've created https://github.com/OpenIDC/pyoidc/issues/365 for that.